### PR TITLE
Cs -186 Redesign the form builder view mode

### PR DESCRIPTION
**Pull request by zsidoaliz**

### CODE REVIEW

File: csatar-plugins/forms/traits/AjaxControllerSimple.php [hunk 1]
```diff
@@ -87,13 +87,15 @@ public function createForm($preview = false)
         $this->widget = new \Backend\Widgets\Form($this, $config);
 
         $this->loadBackendFormWidgets();
-
-        $html = $this->widget->render(['preview' => $preview]);
+        
         if (!$preview){
+            $html = $this->widget->render(['preview' => $preview]);
```
Comment (zsso) [inline csatar-plugins/forms/traits/AjaxControllerSimple.php:92]: $preview will never be true here.

File: csatar-plugins/forms/traits/AjaxControllerSimple.php [hunk 2]
```diff
@@ -87,13 +87,15 @@ public function createForm($preview = false)
...
             $html .= $this->renderValidationTags($record);
         }
-
-        $html .= $this->renderBelongsToManyWithPivotDataAndHasManyRelations($record);
+        else {
+            $html = $this->renderViewMode($this->widget);
+        }
+        $html .= $this->renderBelongsToManyWithPivotDataAndHasManyRelations($record, !$preview);
```
Comment (zsso) [inline csatar-plugins/forms/traits/AjaxControllerSimple.php:98]: *

File: csatar-plugins/forms/traits/AjaxControllerSimple.php [hunk 3]
```diff
@@ -355,7 +524,9 @@ public function onSave()
         $attributeNames = [];
 
         foreach ($config->fields as $key => $value) {
-            $attributeNames[$key] = Lang::get($value['label']);
+            if ($value['type'] !== 'section') {
```
Comment (zsso) [inline csatar-plugins/forms/traits/AjaxControllerSimple.php:527]: Will there be always a $value['type']? 
As I remember this function is used only when added a new relation in edit mode, why do we need this change?

**Review by zsso — APPROVED** [review]

File: csatar-plugins/forms/traits/AjaxControllerSimple.php (hunk 1 above)
Comment (zsidoaliz) [reply csatar-plugins/forms/traits/AjaxControllerSimple.php]: resolved

**Pull request merged**